instructor/templates/instructor/course_modify.html

```diff
@@ -0,0 +1,98 @@
+
+{% extends "instructor/assignment_sidebar.html" %}
+
+{% block right_side %}
+    <script>
+        $(function () {
+            $(".scb_f_course_setup_save_continue_course_button").click(function () {
+                var course_pk = $("option:selected").val();
+                var data = {course_pk: course_pk};
+                if ($(this).attr('name') === 'save') {
+                    data['save'] = 'save';
+                } else {
+                    data['continue'] = 'continue'
+                }
+                var url = '/ab/assignments/course_modify/';
+                $.ajax({
+                    url: url,
+                    type: 'POST',
+                    data: data
+                });
+            });
+        });
+
```

Comment:
Just for convenience. I actually think I can rewrite this functionality in much better way.